README.md

```diff
@@ -1,2 +1,272 @@
 # TIDES
-Transit ITS Data Exchange Specification for historical ITS data
+Transit ITS Data Exchange Specification (TIDES) for historical ITS data.
```

Comment:
Recommended edit: Transit ITS Data Exchange Specification (TIDES) for historical **transit operation** data